flask/app.py

```diff
@@ -0,0 +1,13 @@
+from flask import Flask, render_template, request,json
```

Comment:
Do not import libraries that you do not use. You just need the Flask and render_template for this code.